### DISCUSSION TITLE
Why in function success / error not have to custom message parameter

### DISCUSSION THREAD

**Discussion by rickyviz** [MarcinOrlowski/laravel-api-response-builder#242] [Q&A]
`RB::success($flight);`
in this method, we can't give custom messages,
and also on error

**Comment by MarcinOrlowski** [comment]
These statics are helper methods, so if you have any case that it is not covered, you can easily add own method because what is this helper doing under the hood, is just using RB's builder, so just extend or add own class an build own helpers. As for messages - it's by design where messages are related to API return codes and are pulled automatically (incl. localization support etc). But again, if you have different use case, just build your own helpers atop of current code. You can even do a drop-in replacement by doing `use \Your\Namespace\Class as RB;`. You may want to see [ResponseBuilder.php](https://github.com/MarcinOrlowski/laravel-api-response-builder/blob/master/src/ResponseBuilder.php) as reference on how to create own helpers.

**Reply by rickyviz** [reply]
oke thanks, what diffrent http_code and api_code, i think it same means

**Reply by MarcinOrlowski** [reply]
No, there are two different concepts. See: [docs](https://github.com/MarcinOrlowski/laravel-api-response-builder/blob/master/docs/docs.md) - it's named "Return Codes" there.
